eclipse-templatevariables-feature/feature.xml

```diff
@@ -14,68 +14,68 @@
    </copyright>
 
    <license>
-                                 Apache License
+      Apache License
```

Comment:
any reason why this shall not be centered, while the lines below remain centered?
looks odd to me...

Reply:
Automatically formatted. So no reason.